src/Views/General.vala

```diff
@@ -101,19 +97,45 @@ public class MouseTouchpad.GeneralView : Gtk.Grid {
         hold_scale.width_request = 160;
         hold_scale.add_mark (1.2, Gtk.PositionType.BOTTOM, null);
 
+        var reveal_pointer_switch = new Gtk.Switch ();
+        reveal_pointer_switch.halign = Gtk.Align.START;
+
+        var keypad_pointer_switch = new Gtk.Switch ();
+        keypad_pointer_switch.halign = Gtk.Align.START;
+
+        var keypad_pointer_adjustment = new Gtk.Adjustment (0, 0, 500, 10, 10, 10);
+
+        var pointer_speed_label = new SettingLabel (_("Keypad pointer speed:"));
+        pointer_speed_label.margin_bottom = 7;
+
+        var pointer_speed_scale = new Gtk.Scale (Gtk.Orientation.HORIZONTAL, keypad_pointer_adjustment);
+        pointer_speed_scale.draw_value = false;
+        pointer_speed_scale.add_mark (10, Gtk.PositionType.BOTTOM, null);
+
+        var pointer_speed_help = new Gtk.Label (_("Use a numeric keypad to control the mouse pointer. Note that this disables both levels of keys on your keypad."));
```

Comment:
I feel like this first sentence is redundant with the switch label
```suggestion
        var pointer_speed_help = new Gtk.Label (_("This disables both levels of keys on the numeric keypad."));
```

Reply:
@danrabbit it looks like only you can accept your suggestion since you're the PR author. :stuck_out_tongue: But I agree, and this phrasing better matches the other help text.